docker/Dockerfile.psql

```diff
@@ -0,0 +1,26 @@
+# To build run `docker build -f docker/Dockerfile.psql .` from the root of the
+# zulip repo.
+
+# Install build tools and build tsearch_extras for the current postgres
+# version. Currently the postgres images do not support automatic upgrading of
+# the on-disk data in volumes. So the base image can not currently be upgraded
+# without users needing a manual pgdump and restore.
+FROM postgres:9.5
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y postgresql-server-dev-$PG_MAJOR git build-essential
+RUN git clone https://github.com/zulip/tsearch_extras.git
+RUN cd tsearch_extras && make
```

Comment:
I think if you arranged this differently, you could remove the `postgres-server-dev` and `build-essential` packages in the same `RUN` command where you built it and shrink some unnecessary layers...